app/util.py

```diff
@@ -264,3 +265,17 @@ def create_or_update_borrower_document(
         name=filename,
         verified=verified,
     )
+
+
+def stage_application_as_borrower_accessed_external_onboarding_system(
```

Comment:
The model methods use the wording "stage ..." because we are "staging" changes for commit.

In this method, we are actually doing the commit, so I suggest the name:

```suggestion
def set_borrower_accessed_external_onboarding(
```